### PR TITLE
Lint

### PR DESCRIPTION
**Pull request by SimonSchick**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,22 +32,22 @@
   "dependencies": {
     "lodash.assignin": "^4.0.2",
     "lodash.isplainobject": "^4.0.1",
-    "protobufjs": "^4.0.0",
+    "protobufjs": "^5.0.1",
     "varint": "^4.0.0",
-    "ws": "^0.8.0"
+    "ws": "^1.1.1"
```
Comment (connor4312) [inline package.json]: Makes sure this works on the frontend, ws 1.x drops browser support

**Comment by connor4312** [comment]
works for me

File: package.json (hunk 1 above)
Comment (SimonSchick) [inline package.json]: Well there is this line:

`const WebSocket = isNode ? require('ws') : window.WebSocket;`

Comment (connor4312) [inline package.json]: Okay, so does it work on the frontend? 😛

Comment (SimonSchick) [inline package.json]: Linked this version into local frontend and connected to prod BE, works fine.

Comment (connor4312) [inline package.json]: 👍

**Comment by ProbablePrime** [comment]
lgtm

**Pull request merged**